### PR TITLE
[SYSTEMML-1385] Remove warning from ALS-DS.dml execution

### PR DESCRIPTION
**Pull request by deroneriksson**
Remove initialization warning for loss_init.
Remove redundant comments.
Update input parameter docs to reflect actual default value of lambda param.

### CODE REVIEW

**Comment by deroneriksson** [comment]
Can demonstrate with:
```
import org.apache.sysml.api.mlcontext._
import org.apache.sysml.api.mlcontext.ScriptFactory._
val ml = new MLContext(sc)
val dataGen = dmlFromFile("scripts/datagen/genRandData4ALS.dml")
dataGen.in("$rows", 1000).in("$cols", 1000).in("$rank", 100).in("$nnz", 10000).out("X", "W", "H")
val v = ml.execute(dataGen).getMatrix("X")
val als = dmlFromFile("scripts/algorithms/ALS-DS.dml")
als.in("V", v).in("$rank", 100).out("L", "Rt")
val alsResults = ml.execute(als)
```

**Comment by mboehm7** [comment]
LGTM - as a nice to have, we could be more specific in the description: "ALS algorithm using a direct solve method for individual least squares problems.

File: scripts/algorithms/ALS-DS.dml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 # reg     String   "L2"	    Regularization: 
 #						    "L2" = L2 regularization;
```
Comment (dusenberrymw) [inline scripts/algorithms/ALS-DS.dml]: While you're at it, can you fix the spacing issues here as well?

File: scripts/algorithms/ALS-DS.dml [hunk 2]
```diff
@@ -48,20 +48,20 @@
 # 2- An r x n matrix R
 #
 # HOW TO INVOKE THIS SCRIPT - EXAMPLE:
-# hadoop jar SystemML.jar -f ALS.dml -nvargs V=INPUT_DIR/V L=OUTPUT_DIR/L R=OUTPUT_DIR/R rank=10 reg="L2" lambda=0.0001 fmt=csv
+# hadoop jar SystemML.jar -f ALS-DS.dml -nvargs V=INPUT_DIR/V L=OUTPUT_DIR/L R=OUTPUT_DIR/R rank=10 reg="L2" lambda=0.0001 fmt=csv
```
Comment (dusenberrymw) [inline scripts/algorithms/ALS-DS.dml]: Side thought: We should start replacing `hadoop` with `spark-submit` in all of our scripts.

File: scripts/algorithms/ALS-DS.dml [hunk 3]
```diff
@@ -48,20 +48,20 @@
...
-check      = ifdef ($check, FALSE);	    # $check=FALSE;
-thr        = ifdef ($thr, 0.0001);      # $thr=0.0001;
-fmtO       = ifdef ($fmt, "text");      # $fmt="text";
+r          = ifdef($rank, 10);
+reg        = ifdef($reg, "L2");
+lambda     = ifdef($lambda, 0.000001);
+max_iter   = ifdef($maxi, 50);
+check      = ifdef($check, FALSE);
+thr        = ifdef($thr, 0.0001);
+fmtO       = ifdef($fmt, "text");
```
Comment (dusenberrymw) [inline scripts/algorithms/ALS-DS.dml]: Thanks for fixing the spacing here.  Great example of why tabs shouldn't be used to line things up.

**Comment by deroneriksson** [comment]
Updated ALS-DS description.

Good point about the formatting issues resulting from the combination of tabs and spaces @dusenberrymw. I converted all the tabs to spaces and also removed the trailing whitespace.

I'll merge.

**Pull request closed without merging**